google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/Util.java

```diff
@@ -136,9 +139,9 @@ static Map<String, List<String>> createStatsHeaders(ApiCallContext apiCallContex
     return headers.build();
   }
 
-  static Long getGfeLatency(Metadata metadata) {
-    if (metadata != null && metadata.get(SERVER_TIMING_HEADER_KEY) != null) {
-      String serverTiming = metadata.get(SERVER_TIMING_HEADER_KEY);
+  private static Long getGfeLatency(@Nullable Metadata metadata) {
+    String serverTiming;
+    if (metadata != null && (serverTiming = metadata.get(SERVER_TIMING_HEADER_KEY)) != null) {
```

Comment:
inline assignment is highly discouraged as it's really easy to mistake it for ==

this would be a lot simpler:

if (metadata == null) { return null; }

String serverTiming = metadata.get(SERVER_TIMING_HEADER_KEY);
if (serverTiming == null) {
  return null;
}
...